README.md

```diff
@@ -89,53 +80,42 @@ The `go` method performs three tasks.
    by default, this _runs the first time, and only the first time_, that `go` is invoked
 3. check for new SMS messages and send them to Pngme's system every 30 minutes using an Android Periodic Worker
 
-| var name | description |
-| -------- | ----------- |
-| clientKey | the Pngme SDK Token for your account (see above) |
-| firstName | the mobile phone user's first name |
-| lastName | the mobile phone user's last name |
-| email | the mobile phone user's email address |
-| phoneNumber | the mobile phone user's phone number, example `"23411234567"` |
-| externalId | a unique identifier for the user provided by your app (if none available, pass an empty string `""`)|
-|isKycVerified | a boolean, indicating if your app has verified the user's identity using KYC |
-| companyName | your company's name; this is used in the display header of the permissions UI flow |
+| Field         | Description                                                                                         |
+| ------------- | --------------------------------------------------------------------------------------------------- |
+| clientKey     | the Pngme SDK Token for your account (see above)                                                    |
+| firstName     | the mobile phone user's first name                                                                  |
+| lastName      | the mobile phone user's last name                                                                   |
+| email         | the mobile phone user's email address                                                               |
+| phoneNumber   | the mobile phone user's phone number, example `"23411234567"`                                       |
+| externalId    | a unique identifier for the user provided by your app; if none available, pass an empty string `""` |
+| isKycVerified | a boolean, indicating if your app has verified the user's identity using KYC                        |
+| companyName   | your company's name; this is used in the display header of the permissions UI flow                  |
 
 ### `resetPermissionFlow()`
 
 ```ts
-type resetPermissionFlow = () => void
+type resetPermissionFlow = () => void;
 ```
 
-As noted above, the [Permission Dialog Flow](.docs/permission_flow.gif) will only run the first time that the `go` method is invoked.
-If your app needs to implement logic to show the Dialog Flow again,
-then you can reset the permission flow by calling `resetPermissionFlow`.
-The next time you call `go`, the Permission Dialog Flow will show again.
+The [Permission Dialog Flow](.docs/permission_flow.gif) will only run the first time that the `go` method is invoked.
```

Comment:
These .gif are very helpful. Is there a way to dictate that's what is being linked or could a static flow chart for the details be provided here?

Reply:
Not sure what a good alternative would be but the gifs are already linked in this way elsewhere, so we can scope permission dialog doc improvements separately?